packages/web-components/src/components/content-block-simple/__stories__/content-block-simple.stories.scss

```diff
@@ -0,0 +1,25 @@
+//
+// @license
+//
+// Copyright IBM Corp. 2021
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import 'carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/layout/breakpoint';
+
+@include carbon--breakpoint-down('md') {
+  /* stylelint-disable-next-line selector-type-no-unknown */
+  dds-link-list {
+    /* stylelint-disable-next-line selector-type-no-unknown */
+    dds-link-list-item-card-cta {
+      /* stylelint-disable-next-line selector-type-no-unknown */
+      dds-card-cta-footer {
+        margin-top: 0;
+        display: flex;
+        align-items: center;
+      }
+    }
```

Comment:
@NSDrowned Sorry I missed this in the last review. What happens when this component is used outside Storybook and these styles aren't there? Also, I think these styles are misaligning the icons?

Reply:
@kennylam np... the problem only occurs in this story. 

From my tests they looked perfect outside:

![image](https://user-images.githubusercontent.com/4054756/117188821-b51bd100-ad9a-11eb-8f60-105925288595.png)

I'll take another look at why it's happening.

Reply:
@NSDrowned Oh hm, yes you're right I can see what you mean. Looks like you found a bug and these styles should be added to the codebase 👍